ProvinceMapper/Source/Provinces/Province.h

```diff
@@ -3,17 +3,7 @@
 #include <string>
 #include <vector>
 
-struct Pixel
-{
-	Pixel() = default;
-	explicit Pixel(int tx, int ty, unsigned char tr, unsigned char tg, unsigned char tb);
-	int x = 0;
-	int y = 0;
-	unsigned char r = 0; // these colors vary depending on the state of the province.
-	unsigned char g = 0;
-	unsigned char b = 0;
-};
-
+struct Pixel;
 struct Province
 {
 	Province(int theID, unsigned char tr, unsigned char tg, unsigned char tb, std::string theName);
```

Comment:
I believe ID should be unsigned if we don't expect negative IDs, unless you prefer int for negative sentinels.

Reply:
Naw, I'm the one assigning those links, and unless we start reading lines in reverse, there shouldn't be an issue.
Tis funny though, related. I wanted to make coords to offset foolproof by requiring positive heights and widths and then realized wxWidgets doesn't even bother and is giving me regular ints for coordinates. C'est la vie.

Reply:
Oh, you meant province IDs, those are read from definition.csv. There's no negative stuff in there or the game would have some funny issues.